Client/Ecosia/UI/NTP/Impact/NTPImpactCell.swift

```diff
@@ -27,6 +28,11 @@ final class NTPImpactCell: UICollectionViewCell, Themeable, ReusableCell {
     private var impactRows: [NTPImpactRowView] {
         containerStack.arrangedSubviews.compactMap { $0 as? NTPImpactRowView }
     }
+    private lazy var financialReportCTAView: FinancialReportsCTAView = {
+        let view = FinancialReportsCTAView(actionText: ClimateImpactCTAExperiment.actionText)
+        view.isHidden = !ClimateImpactCTAExperiment.isEnabled
+        return view
+    }()
```

Comment:
We do not add the `financialReportCTAView` to the stack view if the experiment is not enabled as part of the `configureFinancialReportCTAIfNeeded(:)` below.
Wonder whether we can remove the `isHidden` here and make the `financialReportCTAView` a constant initialization.

```suggestion
    private let financialReportCTAView = FinancialReportsCTAView(actionText: ClimateImpactCTAExperiment.actionText)
```